queries/po/highlights.scm

```diff
@@ -0,0 +1,33 @@
+; Keywords
+
+[
+  "msgctxt"
+  "msgid"
+  "msgid_plural"
+  "msgstr"
+  "msgstr_plural"
+] @keyword
+
+; Punctuation
+
+[ "[" "]" ] @punctuation.bracket
+
+; Literals
+
+(string) @string
+
+(escape_sequence) @string.escape
+
```

Comment:
Missing the highlight for format strings.

```suggestion

(string_fragment) @string.special

```

Reply:
Do you mean like %s, %n?

Reply:
Yep.

Reply:
> Yep.

string_fragment captures any string content that isn't an escape sequence, so this doesn't fit. That being said, I could update the grammar to parse those as well, or wait until your format string injections is merged and inject here then.

Reply:
> string_fragment captures any string content that isn't an escape sequence, so this doesn't fit.

Oh, I didn't realise that. Can you try to capture them with `#match?` then?

Reply:
Mmm, sure but string_fragment is a token, how would I match inside that? This seems better suited for your injection (the printf format one)

The rule:

`string_fragment: _ => token.immediate(prec(1, /[^"\\]+/)),`

Reply:
I thought it matched words. Nevermind then.